Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/DbController.cs

```diff
@@ -26,42 +24,44 @@ public DbController(MyDbContext context, IConfiguration config)
             _config = config;
         }
 
-          // GET api/values
+        public DbController()
+        {
+        }
+
+        // GET api/values
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<Entry>>> Get()
+        public async Task<ActionResult<IEnumerable<Users>>> Get()
         {
-            return await _db.Entries.ToListAsync();
+            return await _db.UsersEntries.ToListAsync();
         }
 
         // GET api/values/5
         [HttpGet("{id}")]
         public async Task<IActionResult> GetEntry([FromRoute] int id)
         {
-            var entry = await _db.Entries.SingleOrDefaultAsync(m => m.Id == id);
+            var entry = await _db.UsersEntries.SingleOrDefaultAsync(m => m.UsersId == id);
             if (entry == null)
             {
                 return NotFound();
-
             }
+            _db.UsersEntries.Remove(entry);
             await _db.SaveChangesAsync();
-
             return Ok(entry);
-            
         }
 
         // POST api/values
         [HttpPost]
-        public async Task Post([FromBody] Entry value)
+        public async Task Post([FromBody] Users value)
```

Comment:
Its better to give the parameter a descriptive name like `user` to make it clear inside the function